packages/patrol/test/patrol_tester_test.dart

```diff
@@ -936,5 +937,104 @@ void main() {
         expect($('bottom text').visible, true);
       });
     });
+
+    group('pumpAndMaybeSettle()', () {
+      late int count;
+      late int state;
+
+      setUp(() {
+        count = 0;
+        state = 0;
+      });
+
+      final appWithInfiniteAnimation = MaterialApp(
```

Comment:
I think this widget (and another one below) do not need to be assigned to variables. It's okay to write this code in individual `test()` cases.

See also: [Rule of Three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming))

No strong opinions though.

Reply:
One of this apps is referenced twice, but more important argument would be that it is easier to understand what's going on in test, when you don't have to analyze test app's code.